tests/validation/tests/v3_api/test_v2_cis_scan.py

```diff
@@ -68,10 +78,203 @@ def test_install_v2_cis_benchmark():
     install_v2_app(client, rancher_repo, cis_charts, CHART_NAME, ns)
 
 
+def test_v2_create_clusterscan_permissive():
+    """
+    Create cluster scan with profile: "rke-profile-permissive"
+    Validate the scan CR and the scan report generated
+    """
+    client = \
+        get_cluster_client_for_token_v1(
+            cluster_detail["cluster"]["id"], USER_TOKEN
+        )
+    rancherrepo = \
+        client.list_catalog_cattle_io_clusterrepo(id="rancher-charts")
+    cluster_id = cluster_detail["cluster"]["id"]
+    cluster_name = cluster_detail["cluster"]["name"]
+    rancher_repo = rancherrepo["data"][0]
+
+    # install CIS if not installed
+    if not check_v2_app_installed(client,CHART_NAME):
+        install_cis_v2(client, cluster_id, cluster_name, rancher_repo)
+
+    client = \
+        get_cluster_client_for_token_v1(
+            cluster_detail["cluster"]["id"], USER_TOKEN
+        )
```

Comment:
Why do you get the client again which is the same as line 86 - 89?

Reply:
The first client does not have context to "Apps" which gets installed right before this client statement, because of which my cases were failing. Reload of the client object somehow didnt seem to work. (I am not sure if that is valid)

Reply:
@sowmyav27  It makes sense. Could you add a comment in the code, others may have the same confusion in the future.